spec/unit/provider/package/cab_spec.rb

```diff
@@ -54,6 +54,7 @@
     allow(provider).to receive(:dism_command).with("/Get-Packages").and_return(installed_package_list_obj)
     package_version_obj = double(stdout: package_version_stdout)
     allow(provider).to receive(:dism_command).with("/Get-PackageInfo /PackagePath:\"#{new_resource.source}\"").and_return(package_version_obj)
+    allow_any_instance_of(Chef::Provider::Package::Cab).to receive(:dism_command).with("/Get-PackageInfo /PackagePath:\"#{new_resource.source}\"").and_return(package_version_obj)
```

Comment:
`allow_any_instance_of` is usually codesmell for tests -- can this just be `allow(provider).to receive....`?   although that seems to be the prior line....  i'm somewhat confused as to why this is necessary...